vscode/microsoft-kiota/l10n/bundle.l10n.it.json

```diff
@@ -0,0 +1,42 @@
+{
+  "Client will be upgraded from version {0} to {1}, upgrade your dependencies": "Il client verrà aggiornato dalla versione {0} a {1}, aggiornate le vostre dipendenze",
+  "No description found, select a description first": "Nessuna descrizione trovata, selezionare una descrizione prima",
+  "No endpoints selected, select endpoints first": "Nessun endpoint selezionato, selezionare prima un'API",
+  "No workspace folder found, open a folder first": "Aucun environment de travail sélectionné, ouvrez un environment de travail d'abord",

```

Comment:
This feels very French ;) @mizrael

Reply:
mon Dieu!

Reply:
https://github.com/microsoft/kiota/pull/3511